Documentation/project-docs/developer-guide.md

```diff
@@ -23,12 +23,11 @@ The tasks are represented in scripts (cmd/sh) in the root of the repo:
 * Clean
 * Sync
```

Comment:
Question: Have we defined what Sync will do? if it is just restore everything from external, should we mention that on this doc?